README.md

```diff
@@ -10,43 +10,44 @@
 <br>
 <p align="center">
 Creating test fixtures should be easy.<br>
-<a href="https://www.npmjs.com/package/zod-fixture">zod-fixture</a> helps with the arrange phase of your tests by creating test fixtures based on a <a href="https://github.com/colinhacks/zod">zod</a> schema.
+<a href="https://github.com/timdeschryver/zod-fixture">zod-fixture</a> helps with the arrange phase of your tests by creating test fixtures based on a <a href="https://github.com/colinhacks/zod">zod</a> schema.
 </p>
 
 ## Table of Contents
 
-- [Table of Contents](#table-of-contents)
-- [Installation](#installation)
-- [Getting Started](#getting-started)
-- [Customizing](#customizing)
-  - [Extending](#extending)
-  - [Create Your Own Transformer](#create-your-own-transformer)
-- [API](#api)
-  - [Generators](#generators)
-- [Contributing](#contributing)
-  - [Getting started with GitHub Codespaces](#getting-started-with-github-codespaces)
-  - [StackBlitz](#stackblitz)
-- [Blog posts](#blog-posts)
-- [Credits](#credits)
+[[toc]]
 
 ## Installation
 
-From npm (Node/Bun)
+::: code-group
+
+```sh [npm]
+$ npm install -D vitepress
+```
+
+```sh [pnpm]
+$ pnpm add -D vitepress
+```
+
+```sh [yarn]
+$ yarn add -D vitepress
+```
 
-```bash
-npm install -D zod-fixture
-yarn add -D zod-fixture
-bun add -d zod-fixture
-pnpm add -D zod-fixture
+```sh [bun]
+$ bun add -d zod-fixture
 ```
 
+:::
+
 ## Getting Started
 
 The easiest way to start using zod-fixture is to import the preconfigured `Fixture` class.
 
-<sub>[view implementation](./examples/create-fixture-person.test.ts)</sub>
+<sub>[view implementation](https://github.com/timdeschryver/zod-fixture/tree/beta/examples/create-fixture-person.test.ts)</sub>
 
-```ts
+::: code-group
```

Comment:
This still looks good as a normal markdown file:

![image](https://github.com/timdeschryver/zod-fixture/assets/28659384/08267683-7d52-4913-92dc-c9b41cecb911)